database-posgtres/index.js

```diff
@@ -625,5 +629,49 @@ module.exports = {
     return pg('messages')
       .where({'chatId': chatId})
       .limit(50);
+  },
+
+  createPostG: (userId, text, imageUrl) => {
```

Comment:
Is the G short for something? Maybe better function naming?

Reply:
Fixing - The legacy team had a CreatePost function that I didn't want to delete. Renaming it as createPostById.